File: 0167. Two Sum II - Input array is sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        l = len(numbers)
        a = {}
        for i in range(l):
            sub = target - numbers[i]
            if numbers[i] in a:
                return a[numbers[i]]+1, i+1
            else:
                a[sub] = i

File: 0167. Two Sum II - Input array is sorted.cs
public class Solution {
    public int[] TwoSum(int[] numbers, int target) {
        int left = 0;
        int right = numbers.Length - 1;
        while (numbers[left] + numbers[right] != target)
        {
            if (numbers[left] + numbers[right] > target)
            {
                right--;
            }
            else
            {
                left++;
            }
        }
 
        return new int[] { left + 1, right + 1 };
    }
}
